src/test_l1_dsAudio.c

```diff
@@ -99,6 +100,106 @@ intptr_t null_handle = 0;
 
 /* Global flags to support features */
 static bool extendedEnumsSupported=false; //Default to not supported
+static char* device_type = NULL;
+static char* device_name = NULL;
+static int numports = 0;
```

Comment:
use the module prefix for the variables which are module specific
eg: numports ->gAudioNumberOfPorts
for macros -> DS_AUDIO_XXXX